icu4c/source/common/unicode/utrace.h

```diff
@@ -113,10 +113,35 @@ typedef enum UTraceFunctionNumber {
 #endif  // U_HIDE_DEPRECATED_API
 
 #ifndef U_HIDE_DRAFT_API
+
+    /**
+     * The lowest resource/data location.
+     * @draft ICU 65
+     */
     UTRACE_RES_DATA_START=0x3000,
+
+    /**
+     * Indicates that a value was read from a resource bundle. Provides three
+     * C-style strings to UTraceData: type, file name, and resource path. The
+     * type is "string", "binary", "intvector", "int", or "uint".
+     * @draft ICU 65
+     */
     UTRACE_RESOURCE_PATH=UTRACE_RES_DATA_START,
```

Comment:
In the existing groups, the constants share a prefix, such as UTRACE_UCOL_. How about UTRACE_UDATA_ or UTRACE_DATA_ for these?